src/System.Threading.Tasks.Dataflow/pkg/System.Threading.Tasks.Dataflow.pkgproj

```diff
@@ -8,9 +8,13 @@
 
   <ItemGroup>
     <ProjectReference Include="..\src\System.Threading.Tasks.Dataflow.csproj">
-      <SupportedFramework>net45;netcore45;wp8;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net45;netcore45;wpa81;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Threading.Tasks.Dataflow.WP8.csproj" />
+
+    <!-- WP8 support comes from last-shipped package -->
+    <SupportedFramework Include="wp8">
```

Comment:
This isn't required, but it provides some extra insurance just in case the automatic harvesting doesn't work for some reason.